src/main/scala/dpla/ingestion3/enrichments/StringEnrichments.scala

```diff
@@ -0,0 +1,67 @@
+package dpla.ingestion3.enrichments
+
+import org.apache.commons.lang.StringEscapeUtils
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document.OutputSettings
+import org.jsoup.nodes.Entities.EscapeMode
+import org.jsoup.safety.Whitelist
+
+import scala.util.matching.Regex
+
+/**
+  * String enrichments
+  *
+  */
+class StringEnrichments {
+
+  type SingleStringEnrichment = (String) => String
+  type MultiStringEnrichment = (String) => Array[String]
+
+  /**
+    * Accepts a String value and splits it around periods. Strips
+    * trailing and leading whitespace from those "sentences" and
+    * capitalizes the first character of each sentence leaving all
+    * other characters alone.
+    *
+    * We do not assume that all upper case characters should be
+    * downcased.
+    *
+    */
+  val convertToSentenceCase: SingleStringEnrichment = (value) => {
+    val pattern: Regex = """.*?(\.)""".r
+    val sentences = for( t <- pattern findAllIn value) yield t.trim.capitalize
+    // rejoin the sentences and add back the whitespace that was trimmed off
+    sentences.mkString(" ")
+  }
+
+  val limitCharacters: SingleStringEnrichment = (value) => {
+    ""
+  }
```

Comment:
I'm not sure I understand the use case for `limitCharacters`, but that could be my unfamiliarity with the ingestion process.  In any case, it seems to be the only method without a test.  Does it need one?